### PR TITLE
FIX: revamp Q'kay interface

### LINKED ISSUE #21: Set name of dataset from dataset_description.json

### PR DESCRIPTION
**Pull request by celprov**
enh: display informative error messages to the user
enh: verify that there is at least one HTML file in dataset, otherwise prevent dataset addition.
enh: remove IQMs section from MRIQC report
enh: improve the output log

fix: docker-compose so it takes into consideration .env
fix: make a more intuitive interface to add datasets
fix: do not write templates in the repo
fix: fix bugs related to adding new users
fix: improve visualization of inspection list
fix: remove unused imports, unused files, unused variables and variables that refer to absolute user-dependent path
fix: simplifies html report so it does not need the list + the list length as an input
fix: remove uninformative comments
fix: move out of the main functions related to anonymization, dealing with two folders as they don't seem ready for deployment
sty: ruff

Closes #21, #22, #23, #20

### CODE REVIEW

File: qkay/qkay.py [hunk 1]
```diff
@@ -18,46 +18,64 @@
...
 #     https://www.nipreps.org/community/licensing/
 #
+import base64
+import glob
+import json
+import os
+import random
+
+import numpy as np
+import os.path as op
```
Comment (esavary) [inline qkay/qkay.py:28]: I think we should transition to Pathlib instead of os and glob, but it's better to address this in a future PR.

**Review by esavary — APPROVED** [review]
Looks good to me! I've tested it locally, I think we can merge and proceed from there for future improvements.

File: qkay/qkay.py (hunk 1 above)
Comment (celprov) [reply qkay/qkay.py:28]: Good point!

**Comment by celprov** [comment]
Actually, I don't have the rights to merge this PR, so feel free to do so.

**Pull request merged**